src/Native/Runtime/RuntimeInstance.cpp

```diff
@@ -1741,4 +1744,11 @@ COOP_PINVOKE_HELPER(void *, RhAllocateThunksFromTemplate, (PTR_UInt8 moduleBase,
     return pThunkMap;
 }
 
+#ifdef CORERT
```

Comment:
I would be nice to make this hack touch fewer places, e.g. make it `extern void * g_pDispatchMapTemporaryWorkaround;`. Then it can be just in eetype.inl and bootstrapper.